src/tax/vertex-configure-address.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Configuring Storefront Address Validation
+---
+
+Your [Vertex Cloud account]({% link tax/vertex-account.md %}) also includes the Vertex Address Validation module. When this functionality is enabled for your Magento store, the storefront prompts the customer to correct the address information on both the shipping and billing steps of the one-page checkout.
+
+![]({% link images/images/tax-vertex-cloud-address-verify-store.png %}){: .zoom}
+_Address validation message for correction_
+
+{:.bs-callout-info}
+Using the storefront address validation requires that you first [configure the Vertex Tax Calculations]({% link tax/vertex-configure-magento.md %}) to connect to your Vertex Cloud account.
+
+{:.procedure}
+To enable Vertex address validation for the storefront:
+
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
+
+1. To define the [scope]({% link configuration/scope.md %}) of the configuration for a multisite or multistore installation, set the **Store View** chooser in the upper-left corner to the specific website or store view that is to be configured.
+
+1. In the left panel, expand **Sales** and choose **Address Validation**.
+
+1. Set **Use Vertex Address Validation** to `Enable`.
+
+   This setting enables allows you to configure the address validation settings.
+
+    ![]({% link images/images/config-sales-address-validation.png %}){: .zoom}
+    [_Address Validation settings - enabled_]({% link configuration/sales/address-validation.md %})
+
+1. Verify and accept the default value for **Vertex Address Validation API URL**.
```

Comment:
do we want to say what the default value is, in case it gets changed by accident?

Reply:
If you select the use system value check box it reverts to that.